source/dem/pw_contact_force.cc

```diff
@@ -17,7 +17,9 @@
  * Author: Shahab Golshan, Polytechnique Montreal, 2019
  */
 
+#include <dem/dem_solver_parameters.h>
 #include <dem/pw_contact_force.h>
+#include <fstream>
```

Comment:
Should not be needed there in the long term.

Reply:
How do we exclude it?

Reply:
You just don't include it if you don't need it.